apps/docs-snippets/src/guide/introduction/getting-started.test.ts

```diff
@@ -31,6 +45,7 @@ describe('Getting started', () => {
   });
 
   it('can connect to testnet', async () => {
+    const TESTNET_NETWORK_URL = url;
```

Comment:
This test wasn't mocked before, right?

I think it can continue as it was.

```suggestion
```

Reply:
283c96e33